### PR TITLE
Moving Reward Cards styling out of app

### PR DESCRIPTION
**Pull request by MathieuGilbert**
- Removed unused style rules.
- Adding rule in sections.scss for radio button labels, so the unselected ones aren't white-on-grey.

From:
image
To:
image

🎨

### CODE REVIEW

**Comment by kaytcampbell** [comment]
:book:

File: app/assets/stylesheets/layout_components/sections.scss [hunk 1]
```diff
@@ -201,6 +201,10 @@
     label{
       color: $white;
     }
+
+    input[type="radio"] + label{
+      color: $stormcloud;
+    }
```
Comment (kaytcampbell) [inline app/assets/stylesheets/layout_components/sections.scss]: would this also change the text on the label itself? or just within the radio button?

Comment (MathieuGilbert) [inline app/assets/stylesheets/layout_components/sections.scss]: It only affects labels that are adjacent to radio inputs (not inside):

```
<input type="radio" ....>
<label>gets stormy text color</label>
```

**Comment by kaytcampbell** [comment]
:art: :art: :art:

**Pull request merged**